### PR TITLE
Add TabBarController example and ServiceHolder

### PR DESCRIPTION
**Pull request by zvonicek**

### CODE REVIEW

File: testMVVM/TabBarViewController.swift [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+//  TabBarViewController.swift
+//  MVVMTestProject
+//
+//  Created by Petr Zvoníček on 06.10.16.
+//  Copyright © 2016 Martin Pinka. All rights reserved.
+//
+
+import UIKit
+
+class TabBarViewController: UITabBarController {
```
Comment (mkj-is) [inline testMVVM/TabBarViewController.swift]: This file is not used and named view controller even though it is only controller.

File: testMVVMTests/MVVMTestProjectTests.swift [hunk 2]
```diff
@@ -12,7 +12,11 @@ import PromiseKit
 let mockupSeasons = [Season(name: "blah", episodes: [Episode(name: "episode1"), Episode(name: "episode2"), Episode(name: "episode3")])]
 
 class Test1SeasonServices: SeasonsServices {
-
+    internal func update(episode: Episode, name: String?) -> Promise<Episode> {
+        return Promise { fulfill, reject in
```
Comment (mkj-is) [inline testMVVMTests/MVVMTestProjectTests.swift:16]: Something like `Promise(value: episode)`?

**Review by mkj-is — CHANGES_REQUESTED** [review]
👍

File: testMVVM/TabBarViewController.swift (hunk 1 above)
Comment (zvonicek) [reply testMVVM/TabBarViewController.swift]: Yeah, this should be removed

File: testMVVMTests/MVVMTestProjectTests.swift (hunk 2 above)
Comment (zvonicek) [reply testMVVMTests/MVVMTestProjectTests.swift:16]: Well, this was just a quick fix of already failing test so I used the same solution as in other test cases. This is imho out of the scope of the PR, feel free to improve the test cases.

**Pull request merged**